tecken/base/symboldownloader.py

```diff
@@ -137,8 +168,17 @@ def _get(self, symbol, debugid, filename, refresh_cache=False):
                 source.base_url, self.make_url_path(prefix, symbol, debugid, filename)
             )
             logger.debug(f"Looking for symbol file by URL {file_url!r}")
-            if check_url_head(file_url, _refresh=refresh_cache):
-                return {"url": file_url, "source": source}
+            if last_modified := get_last_modified(file_url, _refresh=refresh_cache):
+                age_days = (datetime.datetime.now(datetime.timezone.utc) - last_modified).days
+                if i == self.try_url_index:
+                    tags = ["symbol_build:try"]
+                else:
+                    tags = ["symbol_build:regular"]
+                metrics.histogram("symboldownloader.file_age_days", age_days, tags)
```

Comment:
The rest of this file emits metrics like `symboldownloader_exists_cache_hit`. This key doesn't follow that convention. That's probably fine given we're probably going to rewrite this code when we do the storage API overhaul. I figured I'd mention it in case you wanted to change it.

Reply:
I did notice the existing pattern, but I think using dots for hierarchy works better in the Grafana UI, and the naming isn't really consistent across the Tecken codebase, so I took the liberty to deviate.

Reply:
I prefer having the hierarchy.

As a side note, when we fix [bug 1867570](https://bugzilla.mozilla.org/show_bug.cgi?id=1867570), it'll be easier to see all the metrics Tecken is emitting and then improve them (fix keys, tags, etc).